app-constants.js

```diff
@@ -23,8 +23,6 @@ const kEnvironmentVariables = [
   "HIBP_STUB_API_ROOT",
   "HIBP_API_ROOT",
   "HIBP_API_TOKEN",
-  "AWS_ACCESS_KEY_ID",
-  "AWS_SECRET_ACCESS_KEY",
```

Comment:
Were these just orphaned or something?

Reply:
Yeah, we don't need them anymore. I removed them from my local `.env` and this keeps throwing errors now. I'll remove these from the heroku environment too as we don't upload anything to AWS anymore.